src/util/fp.js

```diff
@@ -0,0 +1,10 @@
+// Requires a polyfill.
+export const values = obj => Object.values(obj);
+
+export const keys = obj => Object.keys(obj);
+
+export const toPairs = obj => Object.entries(obj);
+
+// This only works with an array. Might implement iterator
+// approach.
+export const map = iteratee => arr => arr.map(iteratee);
```

Comment:
Don't you need a curry helper fn? As we don't want to force people to provide their arguments separately.

Reply:
Good idea. Right now, these aren't used anywhere anyway. I just put them in this module. Future coding 😂 I also want to have a `compose`/`flow` implementation. But then I think most our use cases should be covered.